src/main/java/Duke.java

```diff
@@ -1,10 +1,109 @@
+import java.util.ArrayList;
+import java.util.Scanner;
+
 public class Duke {
+    static ArrayList<Task> list = new ArrayList<>();
+    
+    public static String makeDone(String line) {
+        String msg = "";
+        // sets a specified task as done
+        int index = Integer.valueOf(line.split(" ")[1]);
```

Comment:
Perhaps you can initialize a variable to store line.split so that it can be more clear what it is.